documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,199 @@
+
+.Warm migration fails when cutover conflicts with precopy
+
+In previous releases of {project-short}, the cutover operation failed when it was triggered while precopy was being performed. The VM was locked in {rhv-short} and therefore the `ovirt-engine` rejected the snapshot creation, or disk transfer, operation.
+
+This issue is resolved in {project-short} {project-version}, the cutover operation is triggered, but it is not performed that time because the VM is locked. Once the precopy operation completes, the cutover operation is triggered. link:https://issues.redhat.com/browse/MTV-686[MTV-686]
+
+.Warm migration fails when VM is locked
+
+In previous release of {project-short}, when triggering warm migration while there was an ongoing operation in {rhv-short} that locked the VM. The migration failed because the snapshot creation could not be triggered. 
+
+This issue is resolved in {project-short} {project-version}, warm migration does not fail when an operation that locks the VM is performed in {rhv-short}. The migration does not fail, but starts when the VM is unlocked. link:https://issues.redhat.com/browse/MTV-686[MTV-686] 
+
+
+.Deleting migrated VM does not remove PVC and PV
+
+In previous releases of {project-short}, when removing a VM that was migrated, its persistent volume claims (PVCs) and physical volumes (PV) were not deleted.
+
+This issue is resolved in {project-short} {project-version}, PVC and PV are deleted when deleting migrated VM.link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
+
+.PVC deletion hangs after archiving and deleting migration plan
+
+In previous releases of {project-short}, when a migration failed, its PVCs and PVs were not deleted as expected when its migration plan was archived and deleted.
+
+This issue is resolved in {project-short} {project-version}, PVCs are deleted when archiving and deleting migration plan.link:https://issues.redhat.com/browse/MTV-493[(MTV-493)]
+
+.VM with multiple disks may boot from non-bootable disk after migration
+
+In previous releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target OCP cluster.
+
+This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated are able to boot on the target OCP cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
+
+For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-433?filter=12419160[Resolved Issues] in Jira.
+This release has the following resolved issues:
+
+[id="upgrade-notes-25_{context}"]
+== Upgrade notes
+
+// Is there anything else you want me to add?
+It is recommended to upgrade from {project-short} 2.4.2 to {project-short} {project-version}.
```

Comment:
this week we identified an issue with upgrades from 2.4.1 or 2.4.2 as well.. we need to note here that when upgrading from 2.4 (2.4.z), after the 2.5 operator starts, the mutatingwebhookconfiguration and validatingwebhookconfiguration named forklift-api should be removed:

oc -n openshift-mtv delete mutatingwebhookconfiguration forklift-api
oc -n openshift-mtv delete validatingwebhookconfiguration forklift-api

 and restart the operator